### PR TITLE
SG-34315 - Rebranding

### PR DESCRIPTION
**Pull request by juanburgosautoglb**
Change all the shotgrid/shotgun occurrences in the repository.

### CODE REVIEW

File: engine.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 
 class VREDEngine(sgtk.platform.Engine):
-    """A VRED engine for ShotGrid Toolkit."""
+    """A VRED engine for Flow Production Tracking."""
```
Comment (julien-lang) [inline engine.py:20]:
```suggestion
    """A VRED engine for Flow Production Tracking Toolkit."""
```

File: engine.py [hunk 2]
```diff
@@ -143,24 +144,24 @@ def pre_app_init(self):
             > 0
         ):
             msg = (
-                "The ShotGrid Pipeline Toolkit has not yet been fully tested with VRED {version}. "
-                "You can continue to use the Toolkit but you may experience bugs or "
-                "instability.  Please report any issues you see to {support_url}".format(
+                "The Flow Production Tracking has not yet been fully tested "
```
Comment (julien-lang) [inline engine.py:147]:
```suggestion
                "The Flow Production Tracking Toolkit has not yet been fully tested "
```

File: engine.py [hunk 3]
```diff
@@ -143,24 +144,24 @@ def pre_app_init(self):
...
+                "experience bugs or instability.  Please report any issues you see to {support_url}".format(
                     version=self.vred_version, support_url=sgtk.support_url
                 )
             )
             self.logger.warning(msg)
             if self.has_ui:
                 QtGui.QMessageBox.warning(
                     self._get_dialog_parent(),
-                    "Warning - ShotGrid Pipeline Toolkit!",
+                    "Warning - Flow Production Tracking!",
```
Comment (julien-lang) [inline engine.py:157]:
```suggestion
                    "Warning - Flow Production Tracking Toolkit!",
```

File: engine.py [hunk 4]
```diff
@@ -143,24 +144,24 @@ def pre_app_init(self):
...
                     msg,
                 )
         elif self._version_check(self.vred_version, "2021.0") < 0 and self.get_setting(
             "compatibility_dialog_old_version"
         ):
             msg = (
-                "The ShotGrid Pipeline Toolkit is not fully capable with VRED {version}. "
+                "The Flow Production Tracking is not fully capable with VRED {version}. "
```
Comment (julien-lang) [inline engine.py:164]:
```suggestion
                "The Flow Production Tracking Toolkit is not fully capable with VRED {version}. "
```

File: engine.py [hunk 5]
```diff
@@ -170,7 +171,7 @@ def pre_app_init(self):
             if self.has_ui:
                 QtGui.QMessageBox.warning(
                     self._get_dialog_parent(),
-                    "Warning - ShotGrid Pipeline Toolkit!",
+                    "Warning - Flow Production Tracking!",
```
Comment (julien-lang) [inline engine.py:174]:
```suggestion
                    "Warning - Flow Production Tracking Toolkit!",
```

**Review by julien-lang — CHANGES_REQUESTED** [review]
A few things to change.

Also, I still see one reference to ShotGrid in hooks/tk-multi-publish2/basic/publish_session.py

And what about the plugins/Shotgun/vred_shotgun_menu.gif asset? @staceyoue could you help us with that please?

**Comment by staceyoue** [comment]
> And what about the plugins/Shotgun/vred_shotgun_menu.gif asset? @staceyoue could you help us with that please?

@julien-lang I created this PR to remove obsolete code that used to display the gif - we no longer show that gif, so it will be removed.

**Review by staceyoue — CHANGES_REQUESTED** [review]
To change the "ShotGrid" menu name, you will need to update the text here:
https://github.com/shotgunsoftware/tk-vred/blob/95d8637fa2ddca08f7dd0479fb7b872b201e9e56/python/tk_vred/menu_generation.py#L23

**Review by julien-lang — APPROVED** [review]
LGTM!

**Review by staceyoue — APPROVED** [review]
Awesome!

**Pull request merged**